web/flat-web/src/pages/DevicesTestPage/index.tsx

```diff
@@ -0,0 +1,179 @@
+import "./style.less";
+
+import { DeviceTestPanel } from "flat-components";
+import { observer } from "mobx-react-lite";
+import React, { useCallback, useContext, useEffect, useRef, useState } from "react";
+
+import { DeviceTest } from "../../api-middleware/rtc/device-test";
+import { useParams } from "react-router-dom";
+import { RouteNameType, RouteParams, usePushHistory } from "../../utils/routes";
+import { joinRoomHandler } from "../utils/join-room-handler";
+import { GlobalStoreContext } from "../../components/StoreProvider";
+
+export const DevicesTestPage = observer(function DeviceTestPage() {
+    const globalStore = useContext(GlobalStoreContext);
+    const { roomUUID } = useParams<RouteParams<RouteNameType.JoinPage>>();
+    const pushHistory = usePushHistory();
+
+    const audioResourceSrc =
+        "https://flat-storage.oss-accelerate.aliyuncs.com/cloud-storage/2021-11/25/df1ba448-643c-4b31-8b92-f9439d5d9389/df1ba448-643c-4b31-8b92-f9439d5d9389.mp3";
```

Comment:
use local static. e.g. https://github.com/netless-io/flat/blob/main/desktop/renderer-app/src/assets/media/Goldberg%20Variations%2C%20BWV%20988%20-%2005%20-%20Variatio%204%20a%201%20Clav.mp3

We need to be aware of copyright issues.